src/.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add your IDE-specific folders to `.gitignore` and not commit them to the repository, and certainly not when making a pull request to an upstream repo :-)